README.md

```diff
@@ -64,3 +64,92 @@ Add this project as a dependency in your Package.swift file.
 ```
 .Package(url: "https://github.com/PerfectlySoft/Perfect-PostgreSQL.git", versions: Version(0,0,0)..<Version(10,0,0))
 ```
+
+## QuickStart
+
+Add a file to your project, making sure that it is stored in the Sources directory of your file structure. Lets name it pg_quickstart.swift for example.
+
+Import required libraries:
+```swift
+import PostgreSQL
+```
+
+Setup the credentials for your connection: 
+```swift
+let postgresTestConnInfo = "host=localhost dbname=postgres"
+
+let dataPG = PGConnection()
+```
+
+This function will setup and use a PGConnection
+
+```swift
+public func usePostgres() -> PGResult? {
+    
+    // need to make sure something is available.
+    guard dataPG.connectdb(postgresTestConnInfo) == PGConnection.StatusType.ok else {
+        Log.info(message: "Failure connecting to data server \(postgresTestConnInfo)")
+        
+        return nil
+    }
+
+    defer {
+        dataPG.close()  // defer ensures we close our db connection at the end of this request
+    }
+    // setup basic query
+    //retrieving fields with type name, oid, integer, boolean
+    let queryResult:PGResult = dataPG.exec(statement: "select datname,datdba,encoding,datistemplate from pg_database")
+    
+    defer { queryResult.clear() }
+    return queryResult
+}
+```
+
+Use the queryResult to access the row data using PGRow
+Here is a function that uses several different methods to view the row contents
+
+```swift
+public func useRows(result: PGResult?) {
+    //get rows
+    guard result != nil else {
+        return
+    }
+    
+     for row:PGRow in result! {
+        for f in row {
+            //print field tuples
+            //this returns a tuple (fieldName:String, fieldType:Int, fieldValue:Any?)
+            print("Field: \(f)")
+        }
+     
+     }
+    
+    for row:PGRow in result! {
+    
+        //raw description
+        Log.info(message: "Row description: \(row)")
```

Comment:
hmm, the Log struct is in the PerfectLib, I may not have specified that as an expected dependency. You could use print("Row description: \(row)") if you dont want to import PerfectLib. I get so used to having the many Perfect utilities available that I dont work without it.